presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -83,7 +83,7 @@
         PRESTO_ORC_WRITER_VERSION = version == null ? "UNKNOWN" : version;
     }
 
-    private final SliceOutput output;
+    private final OutputStreamSliceOutput output;
```

Comment:
I thought we were going to change this so the caller passed in an `OutputStream` and we would wrap in this code to a `OutputStreamSliceOutput`